impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -272,8 +272,8 @@ ContextVK::ContextVK(
 
   vk::ApplicationInfo application_info;
   application_info.setApplicationVersion(VK_API_VERSION_1_0);
-  application_info.setApiVersion(VK_API_VERSION_1_0);
-  application_info.setEngineVersion(VK_API_VERSION_1_0);
+  application_info.setApiVersion(VK_API_VERSION_1_1);
+  application_info.setEngineVersion(VK_API_VERSION_1_1);
```

Comment:
The engine version can stay at 1.0. That's the "Flutter Engine" version and the implementation doesn't care about it.